### ISSUE TITLE
hookResponds does not work if response type is JsonReponse or RedirectResponse

### ISSUE THREAD

**Issue by SachinGanesh** [tailflow/laravel-orion#53]
Following response does not trigger **hookReponds**

```php
protected function beforeSave(Request $request, $model)
{
    return response()->json(['message' => 'Invalid data'], 400);
}
```

It is suggested to check response type with  **\Symfony\Component\HttpFoundation\Response** instead of **Illuminate\Http\Response**;

**Comment by SachinGanesh** [comment]
PR https://github.com/tailflow/laravel-orion/pull/54

**Comment by alexzarbn** [comment]
Hi @SachinGanesh,

The PR is merged and published in [v1.2.5 release](https://github.com/tailflow/laravel-orion/releases/tag/1.2.5).
